packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js

```diff
@@ -80,33 +80,18 @@ function getTransformerFactory(context, resourceDir, authConfig) {
         let importedModule;
         const tempModulePath = modulePath.toString();
 
+        console.log('tempModulePath', tempModulePath);
+
         try {
           if (path.isAbsolute(tempModulePath)) {
             // Load it by absolute path
             importedModule = require(modulePath);
           } else {
+            // 'require' from project path
             const projectRootPath = context.amplify.pathManager.searchProjectRootPath();
             const { createRequireFromPath } = require('module');
-            const projectRequire = createRequireFromPath(projectRootPath);
-
-            if (tempModulePath.startsWith('./')) {
```

Comment:
It seems that there is no concept like `global require` and `local require`.
`module.paths` is calculated by caller's path and `NODE_PATH`.
https://stackoverflow.com/questions/15636367/nodejs-require-a-global-module-package